 Javacrash when long press the SSID of connected AP in WiFi list

NFC unavailabilty generates an exception, try/catch solves the issue.

diff --git a/src/com/android/settings/wifi/WifiSettings.java b/src/com/android/settings/wifi/WifiSettings.java
index 0005529..81ec120 100644
--- a/src/com/android/settings/wifi/WifiSettings.java
+++ b/src/com/android/settings/wifi/WifiSettings.java
@@ -484,11 +484,16 @@ public void onCreateContextMenu(ContextMenu menu, View view, ContextMenuInfo inf
                 }
                 if (mSelectedAccessPoint.isSaved()) {
                     menu.add(Menu.NONE, MENU_ID_MODIFY, 0, R.string.wifi_menu_modify);
-                    NfcAdapter nfcAdapter = NfcAdapter.getDefaultAdapter(getActivity());
-                    if (nfcAdapter != null && nfcAdapter.isEnabled() &&
-                            mSelectedAccessPoint.getSecurity() != AccessPoint.SECURITY_NONE) {
-                        // Only allow writing of NFC tags for password-protected networks.
-                        menu.add(Menu.NONE, MENU_ID_WRITE_NFC, 0, R.string.wifi_menu_write_to_nfc);
+                    try {
+                        NfcAdapter nfcAdapter = NfcAdapter.getNfcAdapter(getActivity());
+                        if (nfcAdapter.isEnabled()
+                            && mSelectedAccessPoint.getSecurity() != AccessPoint.SECURITY_NONE) {
+                            // Only allow writing of NFC tags for password-protected networks.
+                            menu.add(Menu.NONE, MENU_ID_WRITE_NFC, 0,
+                                R.string.wifi_menu_write_to_nfc);
+                        }
+                    } catch (UnsupportedOperationException e) {
+                        Log.v(TAG, "this device does not have NFC support");
                     }
                 }
             }
